lib/private/converter.rb

```diff
@@ -3,7 +3,10 @@
 require 'sorbet-runtime'
 require 'polyfill'
 
-using Polyfill(Hash: %w[#slice])
+using Polyfill(
```

Comment:
I think it's easier to remove 2.3 support (in travis.yml) because:
- 2.3 is no longer supported by the ruby community.
- This is a bug in Sorbet-runtime, not sorbet-coerce. We shouldn't polyfill something we're don't use